src/components/Table/Table.react.js

```diff
@@ -26,17 +26,20 @@ export class Table extends React.Component {
       columnHeadings: this.context.tableStore.getColumns(),
       rows: this.context.tableStore.getRows(),
       currentPage: this.context.tableStore.getCurrentPage(),
-      totalPages: this.context.tableStore.getTotalPages()
+      totalPages: this.context.tableStore.getTotalPages(),
+      tableCaption: this.context.tableStore.getTableCaption()
     };
   }
 
   render() {
     var headings = this.state.columnHeadings;
+    var tableCaption = this.state.tableCaption;
 
     return (
       <div className="panel panel-responsive">
         <div className="table-responsive">
           <table className="table table-hover table-striped">
+            <caption>{tableCaption}</caption>
```

Comment:
@andrba no it is not. 
<img width="1239" alt="screen shot 2016-03-10 at 11 28 50 am" src="https://cloud.githubusercontent.com/assets/7036237/13655393/177412b8-e6b3-11e5-83c5-2ed0ad8e6533.png">